source/extensions/quic_listeners/quiche/platform/quic_logging_impl.h

```diff
@@ -31,8 +30,8 @@
       logstream
 
 #define QUIC_LOG_IF_IMPL(severity, condition)                                                      \
-  QUIC_LOG_IMPL_INTERNAL((condition) && quic::IsLogLevelEnabled(quic::severity),                   \
-                         quic::QuicLogEmitter(quic::severity).stream())
+  QUIC_LOG_IMPL_INTERNAL((condition) && IsLogLevelEnabled(severity),                               \
```

Comment:
Can we leave the quic:: as is?  I'm currently trying to

  #define SPDY_LOG_foo  QUIC_LOG_foo

That won't work without the quic:: namespace.

(The bullet point 5 in PR description is not quite right, please update)

Reply:
yes